.gitignore

```diff
@@ -80,6 +80,7 @@ scikits
 !np_datetime_strings.c
 *.c
 *.cpp
+*.pxi
```

Comment:
slightly related question: is there a reason you use `pxi`/`pxi.in` files in this case, while we use `pyx` extension in other cases for included files? (would be nice to do this consistently within pandas)